src/modules/peek/Peek.FilePreviewer/Previewers/ISvgPreviewer.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.UI.Xaml.Media;
+
+namespace Peek.FilePreviewer.Previewers
+{
+    public interface ISvgPreviewer : IPreviewer
```

Comment:
I don't think we need another previewer, we could reuse the IBitmapPreviewer (maybe rename to IImagePreviewer) right and instead of having a BitmapSource we could use a ImageSource?